ASP/HTML-ASP.sublime-syntax

```diff
@@ -14,9 +14,10 @@ contexts:
         - match: "%>"
           scope: punctuation.section.embedded.end.asp
           pop: true
-        - match: (').*?(?=%>)
-          scope: comment.line.apostrophe.asp
-          captures:
-            1: punctuation.definition.comment.asp
-        - include: scope:source.asp
+        - match: ''
+          push:
+            - include: scope:source.asp # don't directly push, so that the scope doesn't also contain 'source.asp ' as well as our 'source.asp.embedded.html '
+          with_prototype:
+            - match: "(?=%>)"
+              pop: true
```

Comment:
I am interested in this, too. While I think it can work well to embed the HTML syntax, one of the challenges (I think) of embedding is that the HTML syntax includes the JavaScript and CSS syntaxes. So it is possible (if for instance the ASP/PHP/etc code is inside of an HTML `<script>` block) that what needs to be embedded is not HTML but JavaScript or CSS.